upi/azure/arm/azlogin.sh

```diff
@@ -0,0 +1,5 @@
+export values=`cat ~/.azure/osServicePrincipal.json`
```

Comment:
hmm, what is the purpose of this file?

Reply:
to allow the login to azure using the values provided by the installer. File is created by installer.

Reply:
removed